modules/virt-deleting-virt-cli.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/install/uninstalling-virt.adoc
+
+[id="virt-deleting-virt-cli_{context}"]
+= Deleting {VirtProductName}
+
+You can delete {VirtProductName} by using the CLI.
+
+.Prerequisites
+
+- Install the OpenShift Command-Line Interface (CLI), commonly known as`oc`.
```

Comment:
The `oc` formatting doesn't render correctly because you're missing a space right before it.

Reply:
Added the missing space.